Design.md

```diff
@@ -162,7 +172,15 @@ Depends on:
 
 ## Keycloak
 
-jboss/keycloak
+Image: `jboss/keycloak`
```

Comment:
This could suggest the latest. Maybe better to specify the tag with the recent version that is supported?